app/controllers/concerns/api/v2/parameter_deprecation.rb

```diff
@@ -0,0 +1,61 @@
+# This module adds a pre-action filter that will handle deprecated properties.
```

Comment:
I disagree, I don't think I'm a novice rails developer and I'm having a hard time understanding this module. We already have a standard mechanism for deprecation warnings as seen in the linked example - `Foreman::Deprecation.api_deprecation_warning`.